tests/libcxx/skipped_tests.txt

```diff
@@ -89,6 +89,9 @@ utilities\format\format.functions\vformat.pass.cpp
 # libc++ doesn't implement P2321R2's changes to vector<bool>::reference
 containers\sequences\vector.bool\iterator_concept_conformance.compile.pass.cpp
 
+# libc++ hasn't updated move_iterator for P2520R0
+std/iterators/predef.iterators/move.iterators/move.iterator/types.pass.cpp

```

Comment:
We have two different files (`expected_results.txt` and `skipped_tests.txt`) describing this test suite for the GitHub and Internal (respectively) test runners. Generally, that means that changes to the two are "mirrored". Changes are never actually identical, however, since (1) this file uses `\` separators and the other `/` separators, (2) paths here are relative to `llvm-project/libcxx/test/std` vs `llvm-project/libcxx/test` for `expected_results.txt`, and (3) this file can only skip tests whereas `expected_results` supports SKIPPED, FAIL, and even PASS (for .fail.cpp tests which don't FAIL) statuses.

TLDR: this line needs to change to:
```suggestion
iterators\predef.iterators\move.iterators\move.iterator\types.pass.cpp
```
and you need to add a section in the same place in `expected_results.txt`:
```c++
# libc++ hasn't updated move_iterator for P2520R0
std/iterators/predef.iterators/move.iterators/move.iterator/types.pass.cpp FAIL
```

Reply:
(Remember yesterday when I mentioned how badly we need to port the GitHub test runner into the internal test infra? This is a lot of the motivation =))